thermo/EquationsOfState.ipynb

```diff
@@ -0,0 +1,414 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Non-ideal equations of state \n",
+    "This example demonstrates the comparison between ideal and non-ideal equations of state (EoS) from Cantera. The following equations of state are used to evaluate thermodynamic properties (enthalpy, entropy, etc.) in this example -\n",
+    "1. Ideal-gas EoS from Cantera\n",
+    "2. Non-ideal Redlich-Kwong EoS from Cantera\n",
+    "3. Helmholtz energy EoS from CoolProp"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Import required packages (Cantera and CoolProp)\n",
+    "CoolProp is an open-source package that contains a highly-accurate database for thermophysical properties. The thermodynamic properties are obtained using pure and pseudo-pure fluid equations of state implemented for 122 components. More details can be found in a reference -\n",
+    "\n",
+    ">I.H. Bell, J.Wronski, S. Quoilin, V. Lemort, “Pure and Pseudo-pure Fluid Thermophysical Property Evaluation and\n",
+    "             the Open-Source Thermophysical Property Library CoolProp,” Industrial & Engineering Chemistry Research 53 (2014), http://pubs.acs.org/doi/pdf/10.1021/ie4033999"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Running Cantera version: 2.5.0b1\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Import Cantera\n",
+    "import cantera as ct\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import time\n",
+    "print('Running Cantera version: ' + ct.__version__)\n",
+    "from CoolProp.CoolProp import PropsSI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Helper functions\n",
+    "This example uses CO$_2$ as the only species. The function 'getThermoCantera' calculates thermodynamic properties based on the thermodynamic state (T, p) of the species using Cantera. Applicable gas-phases are Ideal-gas and Redlich-Kwong. The ideal-gas equation can be stated as\n",
```

Comment:
Is the Redlich Kwong a gas-only phase? If not, the middle sentence here may be misleading.

Reply:
I am not sure if I understand your question correctly. Since the only EoS available currently in Cantera are ideal gas and Redlich-Kwong, the applicable phases for the purpose of this example are ideal-gas and Redlich-Kwong.

Reply:
Right, but the sentence says:

> Applicable _gas-phases_ are Ideal-gas and Redlich-Kwong. 

(emphasis mine). Should it say:

> Applicable phases are Ideal-gas and Redlich-Kwong.

Reply:
I get it now. I have made these changes.